models/br_ms_sinasc/code/br_ms_sinasc_etl.py

```
@@ -0,0 +1,694 @@
+    df = df.rename(columns=RENAME_96_19)
+    df = converter_datas(df)
+    df = converter_hora(df)
+
+    for col_mun in (
+        "id_municipio_nascimento",
+        "id_municipio_mae",
+        "id_municipio_residencia",
+    ):
+        try:
+            df = padronizar_municipio(df, col_mun, id_mun)
+        except Exception as exc:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)